charts/sonarqube-dce/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: sonarqube-dce
 description: SonarQube is a self-managed, automatic code review tool that systematically helps you deliver clean code. As a core element of our Sonar solution, SonarQube integrates into your existing workflow and detects issues in your code to help you perform continuous code inspections of your projects. The tool analyses 30+ different programming languages and integrates into your CI pipeline and DevOps platform to ensure that your code meets high-quality standards.
 type: application
-version: 10.4.0
+version: 10.4.1
```

Comment:
Can you add a changelog also under the `artifacthub.io/changes` [annotation](https://artifacthub.io/docs/topics/annotations/helm/#supported-annotations), please?

Reply:
done